src/css/app.css

```
@@ -1,5 +1,11 @@
 /* app global css */
 
+@media only screen and (min-width: 2090px) and (min-height: 1039px) {
+    #app {
+        transform: scale(1.1);
+    }
```

It looks like the issue is the `body` overflowing. `flex-grow` doesn't seem to respect the new sizes.
https://stackoverflow.com/questions/11065566/body-bigger-than-what-it-should-be
This fixes it:
```suggestion
    }
    body {
        overflow: hidden;
    }
```

That said, I feel that just letting the user zoom manually to 110% is less prone to bugs.